src/sagemaker/xgboost/defaults.py

```diff
@@ -14,7 +14,13 @@
 from __future__ import absolute_import
 
 XGBOOST_NAME = "xgboost"
-XGBOOST_VERSION_1 = "0.90-1"
-XGBOOST_LATEST_VERSION = "0.90-2"
-XGBOOST_SUPPORTED_VERSIONS = [XGBOOST_VERSION_1, XGBOOST_LATEST_VERSION]
+XGBOOST_VERSION_0_90 = "0.90"
+XGBOOST_VERSION_0_90_1 = "0.90-1"
+XGBOOST_VERSION_0_90_2 = "0.90-2"
```

Comment:
I still feel like this constant is unnecessary. Let's ask @ericangelokim for a second opinion.